_posts/2016-09-27-twitter-townhall.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "We've Never Been Silent: %23MyAAPIVote"
```

Comment:
Don't need character encoding here or on lines 10 and 11. Only when the text is going to be spit out into a URL (i.e., for Twitter share links).